tests/test-sessions/test-hazelcast-sessions/src/test/java/org/eclipse/jetty/hazelcast/session/client/ClientModifyMaxInactiveIntervalTest.java

```diff
@@ -30,13 +36,36 @@
     extends AbstractModifyMaxInactiveIntervalTest
 {
 
+    private static final String MAP_NAME = "jetty_foo_session";
+
+    private HazelcastInstance hazelcastInstance;
+
+    @Before
+    public void startHazelcast()
+        throws Exception
+    {
+        Config config = new Config().addMapConfig( new MapConfig().setName( MAP_NAME ) ) //
+            .setInstanceName( "beer" );
+        // start Hazelcast instance
+        hazelcastInstance = Hazelcast.getOrCreateHazelcastInstance( config );
+    }
+
+    @After
+    public void stopHazelcast()
+        throws Exception
+    {
+        hazelcastInstance.shutdown();
+    }
+
     /**
      * @see org.eclipse.jetty.server.session.AbstractTestBase#createSessionDataStoreFactory()
      */
     @Override
     public SessionDataStoreFactory createSessionDataStoreFactory()
     {
         HazelcastSessionDataStoreFactory factory = new HazelcastSessionDataStoreFactory();
+        factory.setOnlyClient( true );
```

Comment:
Looks  bit weird that it is a client, rather than a server. Maybe document what the client setting means on the HazelcastSessionDataStoreFactory?